app/boards/interconnects/pro_micro/pro_micro.zmk.yml

```
@@ -8,3 +8,6 @@ description: |
   The SparkFun Pro Micro grew popular as a low cost ATmega32U4 board with sufficient GPIO and peripherals
   to work for many keyboard needs. Since the original Pro Micro, many pin compatible boards have appeared,
   with various changes or improvements, such as the Elite-C w/ USB-C, nice!nano with nRF52840 wireless.
+  Note: ZMK doesn't support boards with AMR 8-bit processors, such as the ATmega32U4, because Zephyr™ only
```

```suggestion

    Note: ZMK doesn't support boards with AMR 8-bit processors, such as the ATmega32U4, because Zephyr™ only
```

Re-add newline, with indentation.